sdks/v4-sdk/src/utils/v4Planner.ts

```diff
@@ -177,6 +235,9 @@ type RouterAction = {
 }
 
 function createAction(action: Actions, parameters: any[]): RouterAction {
-  const encodedInput = defaultAbiCoder.encode(ABI_DEFINITION[action], parameters)
+  const encodedInput = defaultAbiCoder.encode(
+    V4_SWAP_ABI_DEFINITION[action].map((v) => v.type),
```

Comment:
createAction can be used for more than just "swap" - maybe we call this `V4_BASE_ACTIONS_ABI_DEFINITION` or something?

Reply:
`BaseActions` are shareable across swap and posm

Reply:
fixed!